### DISCUSSION TITLE
Can i still use the good ol' dupe method in sp? (or is there an easier way to get gold?)

### DISCUSSION THREAD

**Discussion by AalphaQ** [diasurgical/DevilutionX#6978] [Q&A] [answered]
Curious as to if I can still drop some gold(or other item), walk a few steps away, click on gold (or other item), and just the same time to click on a potion and have it turn into the gold? 

or if there is a build in method to edit my gold count?

**Comment by Trihedraf** [comment]
No duping has been fixed as it is a bug. DevilutionX isn't a hacking/cheating tool.

**Comment by julealgon** [comment, marked as answer]
Please check existing discussions and issues before posting. This topic has been brought up in several places already, and this was fixed several years ago.

- https://github.com/diasurgical/devilutionX/pull/629
